README.md

```diff
@@ -43,12 +43,15 @@ A collection of awesome things regarding Reason/OCaml ecosystem.
 * [Rexpress](https://github.com/vramana/Rexpress) - Express bindings
 * [reason-js](https://github.com/chenglou/reason-js) - JavaScript bindings via BuckleScript
 
+#### Reason Editor Plugins
+* [vscode-reasonml](https://github.com/freebroccolo/vscode-reasonml) - Reason support for Visual Studio Code 
```

Comment:
The vscode plugin is its own project.  For Vim, Atom, Emacs and Sublime should I just link to the Reason project's [editor support](https://github.com/facebook/reason/tree/master/editorSupport) subprojects?